internal/uvm/create.go

```diff
@@ -66,6 +77,46 @@ type Options struct {
 	ExternalGuestConnection bool
 }
 
+// Verifies that the final UVM options are correct and supported.
+func verifyOptions(ctx context.Context, options interface{}) error {
+	switch opts := options.(type) {
+	case *OptionsLCOW:
+		if opts.EnableDeferredCommit && !opts.AllowOvercommit {
+			return errors.New("Deferred commit is not supported on physically backed VMs")
+		}
+		if opts.SCSIControllerCount > 1 {
+			return errors.New("SCSI controller count must be 0 or 1") // Future extension here for up to 4
+		}
+		if opts.VPMemDeviceCount > MaxVPMEMCount {
+			return fmt.Errorf("vpmem device count cannot be greater than %d", MaxVPMEMCount)
+		}
+		if opts.VPMemDeviceCount > 0 {
+			if opts.VPMemSizeBytes%4096 != 0 {
```

Comment:
Nit: Add space between `%` operator and operands.

Reply:
This is from gofmt and their operator precedence shenanigans. https://play.golang.org/p/jVK9ZhVLeN4 Press format on this and you'll see.